packages/bitcoin/src/__tests__/deposit.test.ts

```diff
@@ -104,7 +110,7 @@ describe(findVaultSwapData, () => {
         boostFee,
         brokerFee,
         retryDuration,
-        affiliates,
+        affiliates: affiliates as Mutable<typeof affiliates>,
```

Comment:
might work as well
```suggestion
        affiliates: affiliates.slice()
```

Reply:
I can try it but I think the objects within have `readonly` fields too

Reply:
the system works